src/bundle/Resources/public/js/scripts/helpers/timezone.helper.js

```diff
@@ -1,22 +1,23 @@
 (function(global, doc, eZ, moment) {
     const userPreferedTimezone = eZ.adminUiConfig.timezone;
-    const userPreferedFullDateFormat = eZ.adminUiConfig.dateFormat.full;
+    const userPreferedLongDateFormat = eZ.adminUiConfig.dateFormat.long;
     const userPreferedShortDateFormat = eZ.adminUiConfig.dateFormat.short;
+    const userPreferedFullDateFormat = eZ.adminUiConfig.dateFormat.long;
```

Comment:
Maybe I put the comment in the wrong place. It's not about missing `userPreferedFullDateFormat` but missing `full` in config here https://github.com/ezsystems/ezplatform-admin-ui/pull/846/files#diff-b682a9ad94862694312dadd69b4a250aL26